fhi_users/models.py

```
@@ -3,32 +3,32 @@
 import datetime
 import typing
 import re
-from django.db import models
+from enum import Enum
+
+from django.db import models, transaction, IntegrityError
```

Ruff (0.8.2)

`django.db.IntegrityError` imported but unused

Remove unused import: `django.db.IntegrityError`

(F401)